client/components/Layout/Layout.tsx

```diff
@@ -67,7 +67,9 @@ const Layout = (props: Props) => {
 					<LayoutPagesCollections
 						content={block.content}
 						pages={communityData.pages}
-						collections={communityData.collections}
+						collections={
+							communityData.collections as DefinitelyHas<Collection, 'title'>[]
```

Comment:
This is another one that comes up quite often: Collections can theoretically not have titles, as the column in the Collection table can be `null`, but in practice it's assumed to be always assigned. Maybe set `@AllowNull{ false }` on the db and set type to `string`?